post-stack.yaml

```diff
@@ -169,6 +169,170 @@
     - include: tasks/devstack/check-devstack-cinder-services.yaml
       when: zuul_project == "openstack/cinder" or zuul_project == "openstack/os-brick"
 
+- hosts: win2016-compute
+  vars_files:
+    - zuul-params.yaml
+    - group_vars/windows
+
+  tasks:
+    - name: Include additional variables based on project
+      include_vars: "job_vars/{{ zuul_project | basename }}.yaml"
+      failed_when: False 
+      tags: always
+
+    - name: Include additional variables based on project job_type
+      include_vars: "job_vars/{{ zuul_project | basename }}-{{ job_type }}.yaml"
+      when: job_type is defined
+      failed_when: False
+      tags: always
+
+    - name: Get neutron_service_check content
+      set_fact:
+        neutron_service_check: "{{ groups['devstack'] | map('extract', hostvars, 'neutron_service_check') | list}}"
+
+    - name: Get nova_service_check content
+      set_fact:
+        nova_service_check: "{{ groups['devstack'] | map('extract', hostvars, 'nova_service_check') | list}}"
+
+    - name: Get cinder_service_check content
+      set_fact:
+        cinder_service_check: "{{ groups['devstack'] | map('extract', hostvars, 'cinder_service_check') | list}}"
+      when: zuul_project == "openstack/cinder" or zuul_project == "openstack/os-brick"
+
+    - block:
+      - name: Set additional variables if network_type == ovs
+        set_fact:
+          neutron_agent_type: "{{ item.name }}"
+          neutron_binary: "{{ item.binary }}"
+          neutron_config: "{{ item.config }}"
+        with_items: "{{ neutron_ovs_agent }}"
+        when: network_type == "ovs"
+
+      - name: Set additional variables if network_type == hyperv
+        set_fact:
+          neutron_agent_type: "{{ item.name }}"
+          neutron_binary: "{{ item.binary }}"
+          neutron_config: "{{ item.config }}"
+        with_items: "{{ neutron_hyperv_agent }}"
+        when: network_type == "hyperv"
+
+      - name: "Check if {{ neutron_agent_type }}.log is present"
+        win_stat:
+          path: "{{ win_dir.log }}\\{{ neutron_agent_type }}.log"
+          get_checksum: False
+        register: neutron_log_stat
+
+      - name: "Run {{ neutron_agent_type }} binary"
+        args:
+          executable: cmd
```

Comment:
Used as executable environment **cmd** so both stdout & stderr to be collected correctly. Using the default executable value **powershell** made not possible this log collection